pkgs/tools/misc/diffoci/default.nix

```
@@ -16,12 +19,30 @@ buildGoModule rec {
 
   vendorHash = "sha256-18rsa91PiqZv70EK3K6K1l6N2mIpoVpkX29amKCo5cg=";
 
-  ldflags = [ "-s" "-w" ];
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=github.com/reproducible-containers/${pname}/cmd/${pname}/version.Version=v${version}"
```

```suggestion
    "-X=github.com/reproducible-containers/diffoci/cmd/diffoci/version.Version=v${version}"
```

See https://github.com/nix-community/nixpkgs-lint/issues/21#issuecomment-1617442720